src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -207,7 +207,7 @@ class PerformInteractionRequest
    * same choice id returns false, otherwise returns
    * true.
    */
-  bool CheckChoiceIDFromRequest(
+  const mobile_apis::Result::eType CheckChoiceIDFromRequest(
```

Comment:
@SKobziev newer forget to fix description if you fix signature. 
Also after your changes make function name not narrative.

Reply:
@LuxoftAKutsan I have updated the function description.